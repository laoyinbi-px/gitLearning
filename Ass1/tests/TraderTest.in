ADD trader1 500
IMPORT trader1 NFL -10
EXPORT trader1 Book 5
EXPORT trader1 Book -10
AMOUNT trader1 Book
IMPORT trader1 pencil 10
EXPORT trader1 pencil 100
exit

