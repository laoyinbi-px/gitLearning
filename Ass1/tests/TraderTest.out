$ Success.
$ Could not import product into market.
$ Could not export product out of market.
$ Could not export product out of market.
$ Product not in inventory.
$ Trader now has 10.00 units of pencil.
$ Could not export product out of market.
$ Have a nice day.
